JuiceMaker/JuiceMaker/Controller/FixStockViewController.swift

```diff
@@ -0,0 +1,69 @@
+
+import UIKit
+
+protocol FixStockViewDelegate {
+    func fixStockView(stock: String)
+}
+
+class FixStockViewController: UIViewController {
+    
+    @IBOutlet weak var strawberryLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var kiwiLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    
+    @IBOutlet weak var strawberryStepper: UIStepper!
+    @IBOutlet weak var bananaStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
+    @IBOutlet weak var kiwiStepper: UIStepper!
+    @IBOutlet weak var mangoStepper: UIStepper!
+   
+    var fruitStore = FruitStore()
+    var delegate: FixStockViewDelegate?
```

Comment:
https://blog.bobthedeveloper.io/swift-retention-cycle-in-closures-and-delegate-836c469ef128